Unified/checkor.py

```diff
@@ -1012,10 +1013,43 @@ def upward( ns ):
 
         time_point("checked custodiality", sub_lap=True)
 
-        ## presence in phedex
+        ## presence in phedex and/or rucio
         phedex_presence ={}
+        rucioClient = RucioClient()
         for output in wfi.request['OutputDatasets']:
-            phedex_presence[output] = phedexClient.getFileCountDataset(url, output )
+            _,dsn,process_string,tier = output.split('/')
+            if tier in UC.get('tiers_to_rucio'):
+                # - creates lists of tuples ot the type: ('blockName', numFiles)
+                #   for all blockNames per Dataset known to both Phedex and Rucio
+                # - creates the union of the two sets in order to avoid any duplicates
+                #   (files present in both systems)
+                # - sums the number of files for the union set
+                # - assigns the value to 'phedex_presence' even though the full sum
+                #   of the files is present in both systems - this way we avoid
+                #   changing the code for the rest of the consistency checks
+                phedex_filecount_pb = phedexClient.getFileCountPerBlock(url, output)
+                rucio_filecount_pb = rucioClient.getFileCountPerBlock(output)
```

Comment:
@todor-ivanov I tried out this function right now and it returned all files instead of a number. I am not sure if that is what you were looking for here. But I know that the rucio client now works for unified.

Reply:
@sharad1126 it should return a list of tuples of the type: ('BlockName', 'Number of files in the block')

Reply:
@todor-ivanov https://github.com/CMSCompOps/WmAgentScripts/pull/561#issuecomment-627410754

Reply:
hmmm , that's strange.... I will check what is going here.

Reply:
@sharad1126 - the function is behaving as it is expected. You are passing a block name instead of a dataset name. You nead:
```
rucioClient.getFileCountPerBlock('/MinBias_TuneCP1_13TeV-pythia8/RunIISummer19UL17SIM-106X_mc2017_realistic_v6-v2/GEN-SIM')
```

instead of 

```
rucioClient.getFileCountPerBlock('/MinBias_TuneCP1_13TeV-pythia8/RunIISummer19UL17SIM-106X_mc2017_realistic_v6-v2/GEN-SIM#2923941b-47fe-42cd-8c9e-9376498cdce1')
```

Reply:
@todor-ivanov is the input to the function expected to be a dataset or a block? Because I gave it a block.